crates/sargon/src/types/samples/account_address_samples.rs

```diff
@@ -0,0 +1,23 @@
+use crate::prelude::*;
+
+pub(crate) static ALL_ACCOUNT_ADDRESSES_SAMPLES: Lazy<[AccountAddress; 10]> =
```

Comment:
hmm this wont work? `ALL_ACCOUNT_SAMPLES.iter().map(|account| account.address()) != ALL_ACCOUNT_ADDRESSES_SAMPLES ` , right? and is not that the point? So `ALL_ACCOUNT_ADDRESSES_SAMPLES` should be removed? and instead:

```diff
impl AccountAddress {
    pub fn sample_at(index: usize) -> Self {
-        ALL_ACCOUNT_ADDRESSES_SAMPLES[index].clone()
+        ALL_ACCOUNT_SAMPLES.iter[index].address()
    }
}
```

right?